client/verta/verta/operations/monitoring/monitored_entity.py

```diff
@@ -77,7 +77,7 @@ def workspace(self):
         self._refresh_cache()
```

Comment:
From working with Heather yesterday, there's a bug where `self._msg.workspace_id` is `0` when the entity is first created; a `find` has to be forced to get the actual ID.
https://github.com/VertaAI/modeldb/pull/2202/files
```suggestion
        self._fetch_with_no_cache()
```